src/Views/AccountView.vala

```diff
@@ -74,6 +75,27 @@ public class Installer.AccountView : AbstractInstallerView {
         confirm_entry_revealer = new ErrorRevealer (".");
         confirm_entry_revealer.label_widget.get_style_context ().add_class (Gtk.STYLE_CLASS_ERROR);
 
+        var hostname_label = new Granite.HeaderLabel (_("Device name")) {
+            margin_top = 16
+        };
+
+        hostname_entry = new Granite.ValidatedEntry () {
+            activates_default = true,
+            hexpand = true
+        };
+
+        hostname_entry.map.connect (() => {
+            hostname_entry.text = Utils.get_hostname ();
+        });
+
+        var hostname_info = new Gtk.Label (_("Visible to other devices when sharing, e.g. with Bluetooth or over the network.")) {
+            max_width_chars = 60,
```

Comment:
Right now this makes the view expand. This will cause the string to wrap as needed without resizing the view:

```suggestion
            // Wrap without expanding the view
            max_width_chars = 0,
```